app/_locales/fr/messages.json

```diff
@@ -2713,15 +2713,15 @@
   },
   "permission_accessNamedSnap": {
     "message": "Se connecter à $1.",
-    "description": "The description for the `wallet_snap_*` permission. $1 is the human-readable name of the snap."
+    "description": "The description for the `wallet_snap` permission. $1 is the human-readable name of the Snap."
   },
   "permission_accessNetwork": {
     "message": "Accéder à internet.",
     "description": "The description of the `endowment:network-access` permission."
   },
   "permission_accessSnap": {
     "message": "Connexion au Snap $1.",
-    "description": "The description for the `wallet_snap_*` permission. $1 is the name of the snap."
+    "description": "The description for the `wallet_snap` permission. $1 is the name of the Snap."
```

Comment:
All of this casing was fixed in this PR: https://github.com/MetaMask/metamask-extension/pull/17915

Looks like you undid that? I think we should revert that.